perimeterx/px_metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+    "version": "1.1.0",
```

Comment:
add instructions to Python Enforcer Release Guidlines to update this file: https://perimeterx.atlassian.net/wiki/spaces/MAI/pages/712671277/Enforcers+Release+Guidelines

Reply:
Great note! Done. :)